lib/scrub.ex

```diff
@@ -48,6 +48,26 @@ defmodule Scrub do
     end
   end
 
+  # Read a nested member. Ordered list should be string split of struct.member.more...
+  def read_tag(session, [tag | _rest] = nested_member) when is_binary(tag) do
+    case Session.get_tag_metadata(session, tag) do
+      {:ok, %{structure: :structured}} ->
+        with {session, conn} <- open_conn(session),
+             data <-
+               ConnectionManager.encode_service(:unconnected_send, request_path: nested_member),
+             {:ok, resp} <- Session.send_unit_data(session, conn, data) do
+          close_conn({session, conn})
```

Comment:
Not required.

Does this always return `:ok`? Idk, might improve readability to pattern match the response here even if its `:ok = `.

Reply:
No, it can also return an error here. It can return {:error, any()}